index.html

```diff
@@ -307,22 +355,15 @@ <h3 class="contact-title">Contact me</h3>
           </li>
           <li>
               <label for="msg"></label>
-              <textarea id="msg" name="user_message" maxlength="500" cols="30" rows="10" required placeholder="Write your message here"></textarea>
+              <textarea class="text-area" name="message" maxlength="500" cols="30" rows="10" required placeholder="Write your message here"></textarea>
         </ul>
```

Comment:
- [ ] [Optional] you have the `Get in touch` buttons please make sure that it follows the `active`, hover, and disable effects.